Fix the kstest report generation

M4rtinK opened the pull request.

jkonecny12 commented on scripts/launcher/lib/virtual_controller.py:
```diff
@@ -303,7 +303,7 @@ def run(self):
         except InstallError as e:
             log.error("ERROR: Image creation failed: %s", e)
-            return False
+            return e
```
I think you want to raise the exception above or resturn just a `str(e)` instead.

jkonecny12 commented on scripts/launcher/run_one_test.py:
```diff
@@ -115,8 +116,10 @@ def run_test(self):
+        try:
+            virt_manager.run()
+        except InstallError as e:
+            self._result_formatter.report_result(False, e)
```
You should call this with `str(e)` instead.

M4rtinK commented:
Updated the PR:
- changed exception propagation as suggested by jkonecny
- handle the two types of tracebacks that show up in the logs in the run report script
